cmd/operator-verify/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	manifests "github.com/operator-framework/api/cmd/operator-verify/manifests"
+
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	rootCmd := &cobra.Command{
+		Use:   "operator-verify",
```

Comment:
I'm fine with that, `operator-verify` was its name when I inherited the project.

@gallettilance @shawn-hurley @awgreene thoughts on naming?

Reply:
I'm actually going to remove this since this repo probably shouldn't be a source of CLI tools. Rather, `operator-sdk` should handle this functionality. At the very least, there will be something like mentioned in the [bundle EP](https://github.com/openshift/enhancements/blob/dec2726b852d7fadd3c7e0419edb24e25ecd67d7/olm/operator-bundle.md#validate-an-operator-bundle).